.appveyor.yml

```diff
@@ -56,3 +56,10 @@ test_script:
       }
 
       & $env:BASH -lc "cd '$env:APPVEYOR_BUILD_FOLDER' && ./hashcat.exe -m 0 --show *ple0.hash"
+
+only_commits:
+  files:
+    - src/**/*
+    - include/*
+    - deps/**/*
+    - Makefile
```

Comment:
Please add OpenCL/inc_* files. Some modules make direct use of them.

Reply:
ok, so I need update also the travis script to don't skip build if OpenCL/inc_* files are changed